src/Tag.php

```diff
@@ -108,21 +108,23 @@ public function __construct(
 	 */
 	private function getTagInstance($tag)
 	{
-		$className = self::getTagClassName($tag);
-		if (class_exists($className)) {
-			return new $className(
-				$this->mpdf,
-				$this->cache,
-				$this->cssManager,
-				$this->form,
-				$this->otl,
-				$this->tableOfContents,
-				$this->sizeConverter,
-				$this->colorConverter,
-				$this->imageProcessor,
-				$this->languageToFont
-			);
-		}
+	  if (preg_match('/^[a-zA-Z]+$/', $tag)) {
+	    $className = self::getTagClassName($tag);
```

Comment:
Please keep indent to tabs: https://github.com/mpdf/mpdf/blob/development/ruleset.xml